main/com.tunein/device.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const Tp = require('thingpedia');
+const axios = require('axios');
+const axiosExtensions = require('axios-extensions');
```

Comment:
Don't use axios. Use the Http helpers in the SDK: https://almond-static.stanford.edu/jsdoc/thingpedia/latest/modules/helpers.http.html
This ensures all skills use the same functionality re authentication, caching, redirects, proxies, etc.